### PR TITLE
Refactor Readme.md 

### PR DESCRIPTION
**Pull request by kailashchoudhary11**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR aims to improve the Readme file and it's readability

- **Why was this change needed?** (You can also link to an open issue here)
The changes are needed because the sentence was not seeming to be grammatically correct.

- **Other information**:

### CODE REVIEW

**Comment by kailashchoudhary11** [comment]
@davidsoderberg Please Review!

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]
I guess we can just use the md syntax for links `[...](...)`

**Comment by kailashchoudhary11** [comment]
> I guess we can just use the md syntax for links `[...](...)`

Thank you for your suggestion! I will change it.

**Review by florian-lefebvre — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The ultimate service for managing multi-channel notifications with a single API.
     ·
     <a href="https://github.com/novuhq/novu/discussions">Request Feature</a>
     ·
-  <a href="https://discord.gg/8KpBEjehEV">Join Our Discord</a>
+  <a href="https://discord.gg/novu">Join Our Discord</a>
```
Comment (p-fernandez) [inline README.md:39]: This breaks the Discord invite. Please rollback.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (scopsy) [reply README.md:39]: Oh seems like my mistake :( I was sure this link works.

We can use created by @iampearceman 
```suggestion
  <a href="https://discord.novu.co">Join Our Discord</a>
```

File: README.md [hunk 2]
```diff
@@ -178,9 +178,8 @@ Novu provides a single API to manage providers across multiple channels with a s
 
 ## 💻  Need Help?
 
-We are more than happy to help you. If you are getting some errors or problems while working with the project, or want to discuss something related to the project.
-
-Just <a href="https://discord.gg/novu">Join Our Discord</a> server and ask for help.
+We are more than happy to help you. If you are getting any error or facing problems while working with the project, then join our 
+[Discord server](https://discord.gg/novu) and ask for help. We are open to discuss anything related to the project.
```
Comment (scopsy) [inline README.md:182]:
```suggestion
[Discord server](https://discord.novu.co) and ask for help. We are open to discuss anything related to the project.
```

**Comment by kailashchoudhary11** [comment]
@scopsy Please Review! Let me know if any changes are required!

**Review by p-fernandez — APPROVED** [review]

**Pull request merged**